src/raftstore/store/peer.rs

```diff
@@ -1086,14 +1089,21 @@ impl Peer {
         self.proposals.push(meta);
     }
 
-    fn get_handle_policy(&mut self, req: &RaftCmdRequest) -> Result<RequestPolicy> {
+    fn get_handle_policy(
+        &mut self,
+        req: &RaftCmdRequest,
+        sync_log: &mut bool,
+    ) -> Result<RequestPolicy> {
         if req.has_admin_request() {
             if apply::get_change_peer_cmd(req).is_some() {
                 return Ok(RequestPolicy::ProposeConfChange);
             }
             if get_transfer_leader_cmd(req).is_some() {
                 return Ok(RequestPolicy::ProposeTransferLeader);
             }
+            if req.get_admin_request().get_cmd_type() == AdminCmdType::Split {
```

Comment:
no need to care ConfChange?

Reply:
it's in propose_conf_change, not propose_normal